### ISSUE TITLE
Failed to reproduce the CIFAR10 result

### ISSUE THREAD

**Issue by zjuchensa** [timlee0212/PENNI#2]
Hi, i run the code on vgg16 and resnet18/CIFAR10, but failed to reproduce the result in your paper, i use the provided baseline and sparse decompose checkpoints, and both failed to achieve your paper results on method-P(after pruning and retraining).

For vgg16 and resnet18, i get 92.75 and 93.41 after pruning and retraining 30 epochs respectively, while in your paper the result is 93.12 and 94.01.

and some other questions:
1) I think the data_prefetcher in your code is wrong, if i directly run your code with the prefetcher, the result is very bad. 
2) There is something wrong with your provided checkpoint resnet56, i cannot load it with torch.load
3) Some details are not consistent with your paper. The cross_interval in your code is 2 and it is 5 in your paper, and SVD is used to solve the decomposition problem while it is PCA in your code.

Thank you!

**Comment by zjuchensa** [comment]
The wrong resnet56 checkpoint is SparseDecompose checkpoint.

**Comment by timlee0212** [comment]
Thank you for pointing out all the issues. The issues 1 have been fixed. 
As for 2, you need to use the dill package. Sorry, I forgot to mention it in the notebook.

For the third question, the parameter in the notebook is not consistent with the paper... Just to demonstrate the usage and you can change it. About the PCA, actually the method described in the paper is PCA (conduct SVD to the covariance matrix). We didn't use the term to avoid confusion since PCA mostly used to analyze the data. 

As for accuracy, I ran the workflow again and got 93.12% and 93.98% for VGG16 and ResNet18. It might be the issue of the training script of the CIFAR10.
